### ISSUE TITLE
no visible difference between selected and unselected actionButton in WHCM 

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#758]
### Expected Behaviour
There should be a visual difference

### Actual Behaviour
The selected and unselected buttons appear the same

#### Steps to Reproduce
Turn on Windows High Contrast Mode
Run Edge/IE/Edge Chromium
Note there is no difference between the selected and not selected actionButton

#### Platform and Version
Windows 10
High Contrast Mode on
Edge Classic / Edge (chromium)

#### Logs taken while reproducing problem
![Spectrum Web Components - Profile 1 - Microsoft​ Edge 7_10_2020 4_40_36 PM](https://user-images.githubusercontent.com/1724479/87210929-32d29580-c2cc-11ea-9b3d-f7ef549d5c7e.png)

**Comment by Westbrook** [comment]
Is this still related to #297 in that this originates upstream? Or have you seen motion of [the issue therein](https://github.com/adobe/spectrum-css/issues/168) that we've yet to adopt?
